File: src/targets/javascript/fetch/fixtures/multipart-form-data-no-params.js
const url = 'https://httpbin.org/anything';
const options = {method: 'POST', headers: {'Content-Type': 'multipart/form-data'}};

try {
  const response = await fetch(url, options);
  const data = await response.json();
  console.log(data);
} catch (error) {
  console.error(error);
}

File: src/targets/rust/reqwest/fixtures/multipart-form-data-no-params.rs
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "https://httpbin.org/anything";

    let form = reqwest::multipart::Form::new()
        .text("", "");

    let client = reqwest::Client::new();
    let response = client.post(url)
        .multipart(form)
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    println!("{}", results);
}
